content/builds/power-deadeye.md

```diff
@@ -0,0 +1,88 @@
++++
+title = "Power Unload Deadeye"
+description = "This build is designed around the skill Unload, being able to refresh Relic of the Thief and being self sufficient in initiative use."
+date = 2024-03-29
+draft = false
+template = "build.html"
+
+[taxonomies]
+tags = ["power","thief","deadeye","pof","lowcog","lowphys","lowrep"]
+authors = ["xellink"]
+
+[extra]
+series = "thief"
+tagline = "Auto-Attack Deadeye with/without Quickness"
+keywords = "Guild Wars 2, GW2, LI, Deadeye"
+toc = true
+spec = "deadeye"
+balance = "March 2024"
++++
+
+The purpose of writing this guide is to provide a very low intensity way to play PvE deadeye. This build is low cognition, low physical complexity, has very few repetitions and is functional in both instanced content. There is an optional amount of APM to push the build to its limit. While this build has a high skill floor, it has a low skill ceiling, requiring daggers and a few button presses to push its damage to near competitive levels.
+At the time of writing, there options for deadeye along the power DPS options does not smoothly transition players from low to high intensity. This simplified build also allows you to perform niche roles like pylon kiting, which is extremely useful when learning the role and not wanting to be bogged down by rotation.
+I have completed all 7 wings in NM with this build being fairly high/average ranked in DPS meters. I do not think that bringing this build is going to gimp your team, but if there are no limitations, there are better options that I hope you would consider as an alternate set to complement this build when you are ready starting with dual daggers. Refer to Section D1.
```

Comment:
"Gimp" is ableist language. Please remove it.

Reply:
i have removed the sentence entirely. I have also redid the entire section and also switched 'hope you would consider' to 'can consider' with regards to the dagger alternative which is of slightly higher complexity.